src/reddit/remove-comment-count.css

```diff
@@ -0,0 +1,27 @@
+/**
+ * Remove comment count
+ * @see B.1
+ */
+
+/* New reddit */
+[id="2x-container"] [data-click-id="comments"] > span
+{
+    text-indent: -9999px;
+    line-height: 0;

```

Comment:
Any reason to avoid the same `font-size: 0;` & `font-size <N>px;` technique you used for old reddit? It gave the same effect when I tried it. All else equal, I have a preference for whichever css disturbs the flow the least.

Reply:
e.g.
```diff
diff --git a/src/reddit/remove-comment-count.css b/src/reddit/remove-comment-count.css
index 1b8310e..8a4a297 100644
--- a/src/reddit/remove-comment-count.css
+++ b/src/reddit/remove-comment-count.css
@@ -6,14 +6,11 @@
 /* New reddit */
 [id="2x-container"] [data-click-id="comments"] > span
 {
-    text-indent: -9999px;
-    line-height: 0;
+    font-size: 0;
 }
 [id="2x-container"] [data-click-id="comments"] > span::after {
     content: "Comments";
-    text-indent: 0;
-    display: block;
-    line-height: initial;
+    font-size: 12px;
 }
```

Reply:
Ah, no: I initially did the old text-indent -9999px trick for both, but it didn't work on the <a> tag, so after some googling I found the font-size:0 method, but I forgot to apply it back to the span. I'll add it.

Reply:
I made the change - though now I think using the "content" pseudo selector is probably not the way to go in the long term... especially for non-english speakers!

Reply:
Yeah.. this project doesn't lend itself well to localizing and testing. The only idea I've had so far for l10n is to do the rules in js, which has obvious trade-offs. That's part of the motivation for the jscss util in this repo.